src/cmd/src/standalone.rs

```diff
@@ -445,11 +447,13 @@ impl StartCommand {
         info!("Standalone options: {opts:#?}");
 
         let mut plugins = Plugins::new();
+
+        let mut plugin_options = opts.plugins;
         let opts = opts.component;
         let fe_opts = opts.frontend_options();
         let dn_opts = opts.datanode_options();
 
-        plugins::setup_frontend_plugins(&mut plugins, &fe_opts)
+        plugins::setup_frontend_plugins(&mut plugins, &fe_opts, &mut plugin_options)
```

Comment:
since `plugin_options` is for plugin setup and the data is from `fe_opts`, can we make it a internal variable of `setup_frontend_plugins`?  Will we still be using it after this `setup_frontend_plugins`?

Reply:
these codes are no longer used